src/CTA.Rules.Models/Actions/ActionValidation.cs

```diff
@@ -4,5 +4,6 @@ public class ActionValidation
     {
         public string Contains { get; set; }
         public string NotContains { get; set; }
+        public string CheckComments { get; set; }
```

Comment:
Can this be changed to a bool? JSON format permits boolean values (and bool properties default to `false` which I believe is the intended behavior here).
Example:
```
{
    "boolField1": true,
    "boolField2": false
}
```